ui/src/overridden/backoffice/Importer/Importer.js

```diff
@@ -0,0 +1,253 @@
+import React, { Component } from 'react';
+import {
+  Container,
+  Header,
+  Segment,
+  Button,
+  Form,
+  Modal,
+  Icon,
+  Label,
+  Message,
+} from 'semantic-ui-react';
+import { invenioConfig } from '@inveniosoftware/react-invenio-app-ils';
+import { ImportedDocuments } from './ImportedDocuments';
+import _isEmpty from 'lodash/isEmpty';
+import _get from 'lodash/get';
+import { http } from '@inveniosoftware/react-invenio-app-ils';
+
+export default class Importer extends Component {
+  constructor(props) {
+    super(props);
+    this.filesRef = React.createRef();
+    this.state = {
+      provider: '',
+      mode: '',
+      file: null,
+      openModal: false,
+      postDone: false,
+      providerMissing: false,
+      modeMissing: false,
+      fileMissing: false,
+      error: {},
+      taskId: null,
+    };
+  }
+
+  handleChange = (e, { name, value }) =>
+    this.setState({
+      [name]: value,
+      providerMissing: false,
+      modeMissing: false,
+      fileMissing: false,
+    });
+
+  postData = async formData => {
+    const importerURL = '/importer';
+    const headers = {
+      headers: {
+        'Content-Type': 'multipart/form-data',
+      },
+    };
+    try {
+      const response = await http.post(`${importerURL}`, formData, headers);
```

Comment:
same here for the endpoint, to have a module in `api` to have all endpoints together.